minio/minio
# Allow site replication config with multiple IDPs

## Description

Fixes a bug that did not let site replication be configured when
multiple IDPs are configured.


## How to test this PR?

Setup multiple IDPs for two minio clusters and try to add site replication configuration. This previously results in a failure, but should work now.

## Types of changes

## Checklist:
